aes/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+project (aes C)
+
+set (SRC host/main.c)
+
+add_executable (${PROJECT_NAME} ${SRC})
+
+include_directories(teec PRIVATE include)
```

Comment:
Doesn't it work with `target_include_directories` instead of `include_directories`? With the former you cut any direct dependencies.